docs/upgradeable_tasks.md

```
@@ -0,0 +1,248 @@
+> - Use `--help` for arguments and flags
+> - Use `--network` to deploy on specific network
+> - These contracts are UPGRADEABLE unlike the clone pattern
+
+### 1. Deploy Test Token (Optional)
+
+If you have an erc20 token contract already deployed, you can skip this step.
+
+For demo purpose, we'll deploy a mock erc20 token KMT.
+
+Run `npx hardhat --network sepolia deploy-mock-token`, you'll get output like:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)